src/OpenTracing/Standard/Format.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace OpenTracing\Standard;
```

Comment:
I think that the builtin formats should be part of OT proper, not `Opentracing\Standard`